.github/workflows/ci.yml

```diff
@@ -1,13 +1,15 @@
 name: Build
 on: [push, pull_request]
+
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: docker://ffurrer/bats:latest
-        with:
-          args: "--recursive ."
+      - uses: actions/checkout@v2
+      - run: |
+          npm install
```

Comment:
using `npm ci` will let your build fail, if the `package-json.lock` is not in sync, which seems to be the case.

Reply:
Right fixed, `npm ci` is a good practice. 

That's strange,  it's not use by default in template : https://github.com/actions/typescript-action/blob/main/.github/workflows/test.yml#L15